src/components/atoms/Popup/Rodal/Dialog/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+
+const Dialog = (props: Props) => {
+  const animation =
+    (props.animationType === 'enter'
+      ? props.enterAnimation
+      : props.leaveAnimation) || props.animation;
+
+
+  const className = `rodal-dialog rodal-${animation}-${props.animationType}`;
+
+  const { width, height, duration, customStyles } = props;
+
+  const style = {
+    width: width,
+    height: height,
+    animationDuration: duration + 'ms',
+    WebkitAnimationDuration: duration + 'ms'
+    // borderTopLeftRadius: '7px',
```

Comment:
remove comment if not required. Keep if it is required at the time of  msite  conversion